src/js/themes/base.js

```diff
@@ -491,9 +494,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     },
     diagram: {
       // extend: undefined,
-      line: {
-        color: 'accent-1',
-      },
+      // line: {
+      //   color: undefined,
```

Comment:
I'll appreciate a clean-up on the base.d.ts file, so `color` will point to ColorType. For some reason, the type is hardcoded to `accent-1`.

Reply:
agreed, done